CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 **********
 
+4.6.2 - 2025-12-18
+******************
+* Fixed params for expiration_date.
+* Add admin table for audit trial.
```

Comment:
NIT: The admin table was added in the last release, these two lines could be changed to something like "Fixed the params for expiration_date in the admin table for audit trial."

Reply:
Thanks. Updated.